src/routes/Inventory.js

```diff
@@ -31,7 +31,7 @@ class Inventory extends Component {
                 <PageHeader>
                     <PageHeaderTitle title='Inventory'/>
                 </PageHeader>
-                <Section type='content'>
+                <Section className="pf-l-page__main-section">
```

Comment:
You can just use the `<Main>` component here

Reply:
See: https://github.com/RedHatInsights/insights-frontend-starter-app/blob/master/src/SmartComponents/SamplePage/SamplePage.js

Reply:
Oh I see!